conf/defaults.ini

```diff
@@ -327,7 +327,7 @@ token_rotation_interval_minutes = 10
 # Set to true to disable (hide) the login form, useful if you use OAuth
 disable_login_form = false
 
-# Set to true to disable the signout link in the side menu. useful if you use auth.proxy
+# Set to true to disable the signout link in the side menu. useful if you use auth.proxy or auth.jwt
```

Comment:
```suggestion
# Set to true to disable the sign out link in the side menu. Useful if you use auth.proxy or auth.jwt.
```

Reply:
Apply all .ini edits to both files.

Reply:
Why does this one not include the default information present in sample.ini? Please reconcile.

Reply:
I think it's normal cause `defaults.ini` and `sample.ini` contents don't always converge.
What I included in sample.ini is not the defaults but it's just an ideas of how the `auth.jwt` could be configured. The only default setting of `auth.jwt` is that it's disabled by default.